[Docs]How to add your custom fixture suites? (#10397)
Sylius/Sylius: docs/customization/suites.rst

                                                PL_WEB:
                                                    amount: 10.00

                    # add your products:
                    mug_product:
                        options:
                            amount: 30

                    sticker_product:
                        options:
                            amount: 20

                    book_product:
                        options:
                            amount: 15

                    tshirt_product:
                        options:
                            amount: 15

                    product_review:
                        options:
                            random: 50

                    similar_product_association:
                        options:
                            amount: 20

                    order:
                        options:
                            amount: 20

                    address:
                        options:
                            random: 10 # the number of addresses that are random created
                            prototype:
                                country_code: PL

> 

Reviewer: I'm missing where we are telling the user to call `bin/console sylius:fixtures:load poland` explicitly

Customizing Suites
==================

What are suites:
~~~~~~~~~~~~~~~~

> Suites are a pocket fixtures that are defined in one place.

Reviewer: ```suggestion
Suites are pocket of fixtures that can be run together.
```

Why would you customize a suites?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
If you need to create a configuration file where you define all necessarily configuration to your store.

How to use suites?
~~~~~~~~~~~~~~~~~~

> List suites are available after ``$ bin/console sylius:fixtures:list`` command.

Reviewer: ```suggestion
Complete list of suites is can be shown with the ``$ bin/console sylius:fixtures:list`` command.
```

> In Sylius default suites are loaded in ``$ bin/console sylius:fixtures:load``.

Reviewer: ```suggestion
In the vanilla Sylius implementation, the ``default` suite is loaded if ``$ bin/console sylius:fixtures:load`` command is executed without any additional argument.
```

> If you creating new suite you must use previous command with name of your suite: ``$ bin/console sylius:fixtures:load your_custom_suite``.

Reviewer: ```suggestion
If you are creating a new suite you must use the previous command with the name of your suite: ``$ bin/console sylius:fixtures:load your_custom_suite``.
```

How to create suites?
~~~~~~~~~~~~~~~~~~~~~

.. tip::

    You can browse the full implementation of this example on `this GitHub Pull Request
    <https://github.com/Sylius/Customizations/pull/24>`__.

> If you want create your fixtures with different locale than en_US you must change parameter in ``src/config/services.yaml``.

Reviewer: ```suggestion
.. warning::
	If you want create your fixtures with different locale than ``en_US`` you must change the ``locale`` parameter in ``src/config/services.yaml``.
	
	.. code-block:: yaml
	    parameters:
	        locale: pl_PL
```

Can you check, how does it look like?

Author: ![Screenshot 2019-08-26 at 10 17 01](https://user-images.githubusercontent.com/39232096/63676194-b1286500-c7ea-11e9-91fb-9ab1e3920725.png)

.. code-block:: yaml

    parameters:
        locale: pl_PL

> Next create ``src/config/packages/fixtures.yaml`` and add below code it there.

Reviewer: ```suggestion
**1.** Create a ``src/config/packages/fixtures.yaml`` file and add below code it there:
```

                                                PL_WEB:
                                                    amount: 10.00

                    # add your products:
                    mug_product:
                        options:
                            amount: 30

                    sticker_product:
                        options:
                            amount: 20

                    book_product:
                        options:
                            amount: 15

                    tshirt_product:
                        options:
                            amount: 15

                    product_review:
                        options:
                            random: 50

                    similar_product_association:
                        options:
                            amount: 20

                    order:
                        options:
                            amount: 20

                    address:
                        options:
                            random: 10 # the number of addresses that are random created
                            prototype:
                                country_code: PL

> Now you must call your custom suite with ``bin/console sylius:fixtures:load poland`` command.

Reviewer: ```suggestion
**2.** Load your custom suite with ``$ bin/console sylius:fixtures:load poland`` command.
```

> Customizing Suites

Reviewer: ```suggestion
Customizing Fixture Suites
```

==================

> What are suites:

Reviewer: ```suggestion
What are suites?
```

~~~~~~~~~~~~~~~~

> Suites are pocket of fixtures that can be run together.

Reviewer: ```suggestion
Suites are pockets of fixtures that can be run together.
```
